docs/sample-code.md

```diff
@@ -0,0 +1,52 @@
+# Python Sample Code for Using Azure OpenAI with API Management
+
+This Python sample demonstrates how to use the Azure OpenAI SDK to interact with an Azure API Management endpoint. The code snippet below outlines the process of initializing the Azure OpenAI client and making a request to generate a response based on the model deployed in your Azure OpenAI service.
+
+## Sample Code
+
+    ```python
+    from openai import AzureOpenAI
+
+    client = AzureOpenAI(
+        azure_endpoint="https://<your_APIM_endpoint>.azure-api.net/<your_api_suffix>", #do not add "/openai" at the end here because this will be automatically added by this SDK
+        api_key="<your subscription key>",
```

Comment:
Is it possible to use managed identity with APIM itself, or only keys?

Reply:
It might be nice to use the best practice here of os.getenv("APIM_SUBSCRIPTION_KEY") to discourage straight up pasting of keys, even though I know they're going to do it anyway. Just a thought.